wordpress-mobile/WordPress-Android
# Update notifications styles

This PR updates the style of the notifications items to the current global spec.

Main changes being made:
- [x] Replace all italic text with bold.
- [x] Adjust avatar and icon sizes.
- [x] Adjust the item's top and bottom margins.
- [x] Adjust the text's colors and sizes.
- [x] Add divider to the item header.
- [x] Make count text bold (e.g. "You've received **5** likes on **Example Blog**").
- [x] Change notification icon border color to match the item background color.
- [ ] ~~Modify unread notification background color (this might not be needed, we still need a confirmation from the design team).~~
- [x] Update FluxC dependency version.

**Internal references:** 
- `pauD4L-z9-p2`
- `pauD4L-z9-p2#comment-502`
- `pauD4L-z9-p2#comment-508`

## To test

Open the notifications tab and make sure the items look like the specs from the references above. Test on dark mode and RTL.

## Notes

- This PR is using the `material-theme` branch to prevent any kind of conflict with existing changes.
- This PR depends on this [FluxC PR](https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/1505) and we'll need to update the FluxC version here after merging it. **Please, do not merge this PR before updating the FluxC dependency version.**

---

PR submission checklist:
